src/engine/components/CharacterController.cpp

```diff
@@ -63,8 +63,9 @@ namespace lei3d
 		m_GroundCheckLocalPos = btVector3(0.f, -height, 0.f);
 
 		m_Collider = new btCapsuleShape(btScalar{ width }, btScalar{ height });
+		m_Collider->setMargin(0.0);
 
-		btScalar  mass{ 1.f };
+		btScalar  mass{ 7.f };
```

Comment:
Can we put these magic numbers into variables?

Reply:
yeah I'll do that in a later pr